src/exercises/exercise-5-power-calculator.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Function that calculates power
+ * @param voltage - The amount of voltage
+ * @param currentPower - The amount of current power
+ * @returns The calculated power
+ */
```

Comment:
Consider renaming the currentPower parameter, I think it's easy to mistake for power instead of current